test cases/common/132 configure file in generator/src/meson.build

```diff
@@ -0,0 +1,7 @@
+compiler = find_program('gen.py')
+gen = generator(compiler,
+  output: '@BASENAME@.h',
+  arguments : ['@INPUT@', '@OUTPUT@'])
+hs = gen.process(cfile)
```

Comment:
Would be nice to also test that a static file via `files()` also works here. Helps check that both files in a subdir of the buildtree (`configure_file()`), and a subdir of the srctree (`files()`) are resolved correctly.